### DISCUSSION TITLE
필요없는 폴더 및 파일 삭제하기

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#250] [Ideas]
배포도 완료됐으니까 필요없는 폴더 및 파일들을 삭제하면 좋을 거 같습니당 !!

**Comment by YuHyun-P** [comment]
동의합니다👍
제출 전까지 해야 할 일들이 필요없는 폴더 및 파일 삭제, ts tsx 확장자 통일화, import 통일화(index.ts), import path alias 통일화 등이 있는 것 같은데 한 사람이 하나의 일을 맡아서 해야 충돌이 없을 것 같네용
